client/components/pages/files/FiltersDialog.vue

```diff
@@ -37,6 +39,16 @@ function reset() {
   resetFilters(localFilters)
 }
 
+function inputDate(event: Event, mapper: (date: Date) => void) {
+  const value = (event.target as HTMLInputElement)?.value
```

Comment:
```suggestion
function inputDate(event: InputEvent, mapper: (date: Date) => void) {
  const value = event.target?.value
```

Reply:
Still need to cast target even with InputEvent..